lib/cf/broker/src/routes/create-service.js

```diff
@@ -19,6 +19,8 @@ const retry = require('abacus-retry');
 const throttle = require('abacus-throttle');
 const breaker = require('abacus-breaker');
 
+const DASHBOARD_URI = (process.env.DASHBOARD_URI || '').endsWith('/') ?
```

Comment:
Most of the constants across Abacus & Broker are camelCase. We might want to stick to this.